.travis.yml

```diff
@@ -63,28 +65,12 @@ branches:
 git:
   depth: 1
 
-# Clones WordPress and configures our testing environment
 before_script:
-    - phpenv config-rm xdebug.ini
-    - export PLUGIN_SLUG=$(basename $(pwd))
-    - source ~/.nvm/nvm.sh
-    - export PATH="$HOME/.composer/vendor/bin:$PATH"
-    - |
-        if [[ ${TRAVIS_PHP_VERSION:0:2} == "7." ]]; then
-            composer global require "phpunit/phpunit=5.7.*"
-        elif [[ ${TRAVIS_PHP_VERSION:0:3} != "5.2" ]]; then
-            composer global require "phpunit/phpunit=4.8.*"
-        fi
-    - nvm install 8
-    - nvm use 8
-    - ./tests/prepare-wordpress.sh
-    - mysql -e "set global wait_timeout = 3600;"
+  - export PLUGIN_SLUG=$(basename $(pwd))
+  - export PATH="$HOME/.composer/vendor/bin:$PATH"
+  - ./tests/setup-travis.sh
 
-script:
-    - echo "Running with the following versions:"
```

Comment:
`phpunit --version` is now part of `run-travis.sh`

Travis does `php -v` for us anyway when it spins up a PHP container:
![image](https://user-images.githubusercontent.com/87168/45807325-ecaddb80-bccb-11e8-85ab-f0552dabb3b2.png)